spec/pods.md

```diff
@@ -159,7 +159,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
     * **app** (object, optional) substitute for the app object of the referred image's ImageManifest. See [Image Manifest Schema](aci.md#image-manifest-schema) for what the app object contains.
     * **mounts** (list of objects, optional) list of mounts mapping an app mountPoint to a volume. Each mount has the following set of key-value pairs:
       * **volume** (string, required) name of the volume that will fulfill this mount (restricted to the [AC Name](types.md#ac-name-type) formatting)
-      * **mountPoint** (string, required) name of the app mount point to place the volume on (restricted to the [AC Name](types.md#ac-name-type) formatting)
+      * **path** (string, required) path inside of the app filesystem to mount the volume; generally this will come from one of an apps mountPoint paths
```

Comment:
Right, the basic observation is two fold:

1) We want to enable volumes to be mounted anywhere
2) We want to give applications the ability to have a descriptive name of the mount points so a user can guess/discover where to place a relevant volume

But, the spec doesn't need to enforce a mapping between volume and mount point as it did before.